itsim/datastore/datastore.py

```diff
@@ -21,108 +21,104 @@ def __init__(self, **kwargs):
         pass
 
     @abstractmethod
-    def load_item(self, item_type: str, uuid: str, from_time: str = None, to_time: str = None) -> Any:  # get
-        pass
-
-    @abstractmethod
-    def store_item(self, sim_uuid: str, data: Any, overwrite: bool = True) -> int:  # post
+    # get
+    def load_item(self, item_type: str, uuid: str, from_time: str = None, to_time: str = None) -> Tuple[str, int]:
         pass
 
     @abstractmethod
-    def create_table(self, table_name, column_list):
+    # post
+    def store_item(self, data: Any, overwrite: bool = True) -> str:
         pass
 
     @abstractmethod
     def delete(self, item_type, uuid):
         pass
 
 
-class DatastoreLocalClient(DatastoreClient):
+class DatastoreRestClient(DatastoreClient):
 
-    def __init__(self, **kwargs) -> None:
-        if kwargs['type'] == 'sqlite':
-            self._database = DatabaseSQLite(kwargs['sqlite_file'])
+    def __init__(self, hostname: str = '0.0.0.0', port: int = 5000, sim_uuid: UUID = uuid4()) -> None:
+        self._hostname = hostname
+        self._port = port
+        self._sim_uuid = sim_uuid
+        self._headers = {'Accept': 'application/json'}
+        self._db_file = '.sqlite'
```

Comment:
If this is meant to be a temporary file, we should use `tempfile.mkstemp`.

Reply:
We must remove this path ourselves on destruction. Here is a suggestion for a real temp file setup:
```suggestion
        self._db_file = tempfile.mkstemp(suffix=".sqlite")
```

Reply:
Interesting, I wasn't aware of this!